index.html

```diff
@@ -22,10 +22,151 @@ <h1 class="myName">
                     Amr Ahmed
                 </h1>
                 <h3 class="role">Software Developer</h3>
-                <p class="self-intro">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
+                <p class="self-intro">I can help you build a product , feature or website Look tdrough some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
             </section>
+
```

Comment:
- [ ] **REQUIRED:** To write better and clean code, have your semantic HTML element (header, section, and so on...) outside of a div tag, not the opposite. For example:
```
<header>
    <div></div>
<header>

<section>
   <div></div>
<section>